Resources/Prototypes/DeltaV/Entities/Mobs/Player/rodentia.yml

```diff
@@ -0,0 +1,34 @@
+- type: entity
```

Comment:
this and BaseMobRodentia yml have huge duplication with BaseSpeciesOrganic, remove everything thats inherited